### PR TITLE
[zh]: Resync kube-scheduler references files.

### PR DESCRIPTION
**Pull request by HongjiangHuang**
Resync [v1.21](https://github.com/kubernetes/website/issues/27673) kube-scheduler reference

### CODE REVIEW

File: content/zh/docs/reference/command-line-tools-reference/kube-scheduler.md [hunk 1]
```diff
@@ -58,10 +60,11 @@ If true, adds the file directory to the header of the log messages
 <tr>
 <td></td><td style="line-height: 130%; word-wrap: break-word;">
 <!-- 
-DEPRECATED: the IP address on which to listen for the --port port (set to 0.0.0.0 for all IPv4 interfaces and :: for all IPv6 interfaces). See --bind-address instead. 
+DEPRECATED: the IP address on which to listen for the --port port (set to 0.0.0.0 or :: for listening in all interfaces and IP families). See --bind-address instead. This parameter is ignored if a config file is specified in --config.
 -->
-已弃用: 要监听 --port 端口的 IP 地址（对于所有 IPv4 接口设置为 0.0.0.0，对于所有 IPv6 接口设置为 ::）。 
+已弃用: 要监听 --port 端口的 IP 地址（将其设置为 0.0.0.0 或者 :: 用于监听所有 接口 和 IP族）。
```
Comment (tengqm) [inline content/zh/docs/reference/command-line-tools-reference/kube-scheduler.md:65]:
```suggestion
已弃用: 要监听 --port 端口的 IP 地址（将其设置为 0.0.0.0 或者 :: 用于监听所有接口和 IP族）。 
```

File: content/zh/docs/reference/command-line-tools-reference/kube-scheduler.md [hunk 2]
```diff
@@ -78,14 +81,26 @@ DEPRECATED: the scheduling algorithm provider to use, this sets the default plug
...
 </tr>
 
+<tr>
+<td colspan="2">--allow-metric-labels stringToString&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Default: []</td>
+</tr>
+<tr>
+<td></td><td style="line-height: 130%; word-wrap: break-word;">
+<!--
+The map from metric-label to value allow-list of this label. The key's format is &lt;MetricName&gt;,&lt;LabelName&gt;. The value's format is &lt;allowed_value&gt;,&lt;allowed_value&gt;...e.g. metric1,label1='v1,v2,v3', metric1,label2='v1,v2,v3' metric2,label1='v1,v2,v3'.
+-->
```
Comment (tengqm) [inline content/zh/docs/reference/command-line-tools-reference/kube-scheduler.md:91]: 未翻译？

File: content/zh/docs/reference/command-line-tools-reference/kube-scheduler.md [hunk 3]
```diff
@@ -203,12 +218,12 @@ Path to the file containing Azure container registry configuration information.
...
 
 <tr>
-<td colspan="2">--bind-address ip&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<!--Default:-->默认值：0.0.0.0</td>
+<td colspan="2">--bind-address string&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<!--Default:-->默认值：0.0.0.0</td>
 </tr>
 <tr>
 <td></td><td style="line-height: 130%; word-wrap: break-word;">
 <!--
-The IP address on which to listen for the --secure-port port. The associated interface(s) must be reachable by the rest of the cluster, and by CLI/web clients. If blank, all interfaces will be used (0.0.0.0 for all IPv4 interfaces and :: for all IPv6 interfaces).
+The IP address on which to listen for the --secure-port port. The associated interface(s) must be reachable by the rest of the cluster, and by CLI/web clients. If blank or an unspecified address (0.0.0.0 or ::), all interfaces will be used.
```
Comment (tengqm) [inline content/zh/docs/reference/command-line-tools-reference/kube-scheduler.md:226]: 230 行译文要调整，与这里更改后的英文不一致。

File: content/zh/docs/reference/command-line-tools-reference/kube-scheduler.md [hunk 4]
```diff
@@ -248,7 +263,7 @@ If set, any request presenting a client certificate signed by one of the authori
 <tr>
 <td></td><td style="line-height: 130%; word-wrap: break-word;">
 <!--
-The path to the configuration file. The following flags can overwrite fields in this file:<br/>  --address<br/>  --port<br/>  --use-legacy-policy-config<br/>  --policy-configmap<br/>  --policy-config-file<br/>  --algorithm-provider
+The path to the configuration file. The following flags can overwrite fields in this file:<br/>--algorithm-provider<br/>--policy-config-file<br/>--policy-configmap<br/>--policy-configmap-namespace
```
Comment (tengqm) [inline content/zh/docs/reference/command-line-tools-reference/kube-scheduler.md:266]: 269-274 行的译文要调整。这里的列表变化了。

File: content/zh/docs/reference/command-line-tools-reference/kube-scheduler.md [hunk 5]
```diff
@@ -261,17 +276,29 @@ The path to the configuration file. The following flags can overwrite fields in
 </tr>
 
 <tr>
-<td colspan="2">--contention-profiling</td>
+<td colspan="2">--contention-profiling&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Default: true</td>
```
Comment (tengqm) [inline content/zh/docs/reference/command-line-tools-reference/kube-scheduler.md:279]:
```suggestion
<td colspan="2">--contention-profiling&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<!--Default:-->默认值：true</td>
```

File: content/zh/docs/reference/command-line-tools-reference/kube-scheduler.md [hunk 6]
```diff
@@ -261,17 +276,29 @@ The path to the configuration file. The following flags can overwrite fields in
...
 </tr>
 <tr>
 <td></td><td style="line-height: 130%; word-wrap: break-word;">
 <!--
-DEPRECATED: enable lock contention profiling, if profiling is enabled
+DEPRECATED: enable lock contention profiling, if profiling is enabled. This parameter is ignored if a config file is specified in --config.
 -->
 已弃用: 如果启用了性能分析，则启用锁竞争分析
```
Comment (tengqm) [inline content/zh/docs/reference/command-line-tools-reference/kube-scheduler.md:286]: 译文没有包含新增的一句话。

**Review by tengqm — COMMENTED** [review]
Just saw that this one is still a WIP.
Hope the initial comments are still useful.

**Comment by HongjiangHuang** [comment]
> Just saw that this one is still a WIP.
> Hope the initial comments are still useful.

Yes, the WIP is because I want to make my work easier through the GitHub diff board.
Now, that the work is done.
Thanks for your review.

File: content/zh/docs/reference/command-line-tools-reference/kube-scheduler.md (hunk 1 above)
Comment (HongjiangHuang) [reply content/zh/docs/reference/command-line-tools-reference/kube-scheduler.md:65]: 这里接口和IP族 似乎都是一个特殊名词. 不需要空格嘛。
另外有个疑问. 结合上下文这里的接口应该是指 "网络接口", 不知道 译成 "网络接口" 对读者会不会更加友好

Comment (tengqm) [reply content/zh/docs/reference/command-line-tools-reference/kube-scheduler.md:65]: 约定在中文与英文（数字）之间留白，并不是为特殊/专有名字留白。
在这个上下文中，接口一词的含义比较清楚，不必译为网络接口。译为网络接口也是可以的。

File: content/zh/docs/reference/command-line-tools-reference/kube-scheduler.md [hunk 7]
```diff
@@ -78,14 +81,31 @@ DEPRECATED: the scheduling algorithm provider to use, this sets the default plug
...
+<tr>
+<td colspan="2">--allow-metric-labels stringToString&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
+<!--Default-->默认值: []</td>
+</tr>
+<tr>
+<td></td><td style="line-height: 130%; word-wrap: break-word;">
+<!--
+The map from metric-label to value allow-list of this label. The key's format is &lt;MetricName&gt;,&lt;LabelName&gt;. The value's format is &lt;allowed_value&gt;,&lt;allowed_value&gt;...e.g. metric1,label1='v1,v2,v3', metric1,label2='v1,v2,v3' metric2,label1='v1,v2,v3'.
+-->
+这个键值映射表设置 资源标签 所允许设置的值。
```
Comment (tengqm) [inline content/zh/docs/reference/command-line-tools-reference/kube-scheduler.md:93]:
```suggestion
这个键值映射表设置度量值标签可接受的值。
```

File: content/zh/docs/reference/command-line-tools-reference/kube-scheduler.md [hunk 8]
```diff
@@ -78,14 +81,31 @@ DEPRECATED: the scheduling algorithm provider to use, this sets the default plug
...
+其中 键 的格式是 &lt;MetricName&gt;,&lt;LabelName&gt;。
```
Comment (tengqm) [inline content/zh/docs/reference/command-line-tools-reference/kube-scheduler.md:94]:
```suggestion
其中键的格式是 &lt;MetricName&gt;,&lt;LabelName&gt;。
```

File: content/zh/docs/reference/command-line-tools-reference/kube-scheduler.md [hunk 9]
```diff
@@ -78,14 +81,31 @@ DEPRECATED: the scheduling algorithm provider to use, this sets the default plug
...
+值 的格式是 &lt;allowed_value&gt;,&lt;allowed_value&gt;。
```
Comment (tengqm) [inline content/zh/docs/reference/command-line-tools-reference/kube-scheduler.md:95]:
```suggestion
值的格式是 &lt;allowed_value&gt;,&lt;allowed_value&gt;。
```

File: content/zh/docs/reference/command-line-tools-reference/kube-scheduler.md [hunk 10]
```diff
@@ -203,16 +223,17 @@ Path to the file containing Azure container registry configuration information.
...
 <tr>
 <td></td><td style="line-height: 130%; word-wrap: break-word;">
 <!--
-The IP address on which to listen for the --secure-port port. The associated interface(s) must be reachable by the rest of the cluster, and by CLI/web clients. If blank, all interfaces will be used (0.0.0.0 for all IPv4 interfaces and :: for all IPv6 interfaces).
+The IP address on which to listen for the --secure-port port. The associated interface(s) must be reachable by the rest of the cluster, and by CLI/web clients. If blank or an unspecified address (0.0.0.0 or ::), all interfaces will be used.
 -->
 监听 --secure-port 端口的 IP 地址。
 集群的其余部分以及 CLI/ Web 客户端必须可以访问关联的接口。
 如果为空，将使用所有接口（0.0.0.0 表示使用所有 IPv4 接口，"::" 表示使用所有 IPv6 接口）。
+如果为空或未指定地址 (0.0.0.0 或 ::)，所有接口将被使用
```
Comment (tengqm) [inline content/zh/docs/reference/command-line-tools-reference/kube-scheduler.md:236]:
```suggestion
如果为空或未指定地址 (0.0.0.0 或 ::)，所有接口将被使用。
```

File: content/zh/docs/reference/command-line-tools-reference/kube-scheduler.md [hunk 11]
```diff
@@ -248,27 +269,42 @@ If set, any request presenting a client certificate signed by one of the authori
...
+
+<tr>
+<td colspan="2">--contention-profiling&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<!--Default-->默认值: true</td>
+</tr>
+<tr>
+<td></td><td style="line-height: 130%; word-wrap: break-word;">
+<!--
+DEPRECATED: enable lock contention profiling, if profiling is enabled. This parameter is ignored if a config file is specified in --config.
+-->
+已弃用: 如果启用了性能分析，则启用锁竞争分析，如果 --config 指定了一个配置文件，那么这个参数将被忽略。
```
Comment (tengqm) [inline content/zh/docs/reference/command-line-tools-reference/kube-scheduler.md:294]:
```suggestion
已弃用: 如果启用了性能分析，则启用锁竞争分析。如果 --config 指定了一个配置文件，那么这个参数将被忽略。
```

File: content/zh/docs/reference/command-line-tools-reference/kube-scheduler.md [hunk 12]
```diff
@@ -248,27 +269,42 @@ If set, any request presenting a client certificate signed by one of the authori
...
+<td colspan="2">--disabled-metrics strings</td>
 </tr>
 <tr>
 <td></td><td style="line-height: 130%; word-wrap: break-word;">
 <!--
-DEPRECATED: enable lock contention profiling, if profiling is enabled
+This flag provides an escape hatch for misbehaving metrics. You must provide the fully qualified metric name in order to disable it. Disclaimer: disabling metrics is higher in precedence than showing hidden metrics.
 -->
-已弃用: 如果启用了性能分析，则启用锁竞争分析
+这个标志提供了一个规避不良指标的入口。你必须提供完整的指标名称，用于禁用它。
```
Comment (tengqm) [inline content/zh/docs/reference/command-line-tools-reference/kube-scheduler.md:306]:
```suggestion
这个标志提供了一个规避不良指标的选项。你必须提供完整的指标名称才能禁用它。
```

File: content/zh/docs/reference/command-line-tools-reference/kube-scheduler.md [hunk 13]
```diff
@@ -248,27 +269,42 @@ If set, any request presenting a client certificate signed by one of the authori
...
+免责声明：禁用指标的优先级比显示和隐藏的指标更高。
```
Comment (tengqm) [inline content/zh/docs/reference/command-line-tools-reference/kube-scheduler.md:307]:
```suggestion
免责声明：禁用指标的优先级比显示隐藏的指标更高。
```

File: content/zh/docs/reference/command-line-tools-reference/kube-scheduler.md [hunk 14]
```diff
@@ -287,7 +323,7 @@ DEPRECATED: enable lock contention profiling, if profiling is enabled
 </tr>
 
 <tr>
-<td colspan="2">--feature-gates mapStringBool</td>
+<td colspan="2">--feature-gates &lt;comma-separated 'key=True|False' pairs&gt;</td>
```
Comment (tengqm) [inline content/zh/docs/reference/command-line-tools-reference/kube-scheduler.md:326]:
```suggestion
<td colspan="2">--feature-gates &lt;<!--comma-separated 'key=True|False' pairs-->逗号分隔的 'key=True|False' 对&gt;</td>
```

File: content/zh/docs/reference/command-line-tools-reference/kube-scheduler.md [hunk 15]
```diff
@@ -744,14 +803,19 @@ Maximum number of seconds between log flushes
...
+--stderrthreshold, --vmodule, --log-flush-frequency.<br/>
+Non-default choices are currently alpha and subject to change without warning.
 -->
-设置日志格式。可选格式："json"、"text"。<br/>
-非默认格式不会在意以下标志设置：
---add_dir_header、--alsologtostderr、--log_backtrace_at、--log_dir、
---log_file、--log_file_max_size、--logtostderr、--one_output、
---skip_headers、--skip_log_headers、--stderrthreshold、--vmodule、
---log-flush-frequency.<br/>
+设置日志格式。可选格式：&quot;json&quot;、&quot;text&quot;。<br/>
```
Comment (tengqm) [inline content/zh/docs/reference/command-line-tools-reference/kube-scheduler.md:813]: 引号不必转义。

```suggestion
设置日志格式。可选格式：“json”、“text”。<br/>
```

File: content/zh/docs/reference/command-line-tools-reference/kube-scheduler.md [hunk 16]
```diff
@@ -744,14 +803,19 @@ Maximum number of seconds between log flushes
...
+非默认格式以下标识不会生效：
```
Comment (tengqm) [inline content/zh/docs/reference/command-line-tools-reference/kube-scheduler.md:814]:
```suggestion
采用非默认格式时以下标志不会生效：
```

File: content/zh/docs/reference/command-line-tools-reference/kube-scheduler.md [hunk 17]
```diff
@@ -786,22 +850,38 @@ Kubernetes API 服务器的地址（覆盖 kubeconfig 中的任何值）。
...
 
+<tr>
+<td colspan="2">--permit-address-sharing</td>
+</tr>
+<tr>
+<td></td><td style="line-height: 130%; word-wrap: break-word;">
+<!--
+If true, SO_REUSEADDR will be used when binding the port. This allows binding to wildcard IPs like 0.0.0.0 and specific IPs in parallel, and it avoids waiting for the kernel to release sockets in TIME_WAIT state. [default=false]
+-->
+如果为 true，在绑定端口时将使用SO_REUSEADDR。
```
Comment (tengqm) [inline content/zh/docs/reference/command-line-tools-reference/kube-scheduler.md:867]:
```suggestion
如果为 true，在绑定端口时将使用 SO_REUSEADDR。
```

File: content/zh/docs/reference/command-line-tools-reference/kube-scheduler.md [hunk 18]
```diff
@@ -786,22 +850,38 @@ Kubernetes API 服务器的地址（覆盖 kubeconfig 中的任何值）。
...
+这将允许通配符IP比如(0.0.0.0)和指定ip一起并行绑定，
```
Comment (tengqm) [inline content/zh/docs/reference/command-line-tools-reference/kube-scheduler.md:868]:
```suggestion
这将允许同时绑定 0.0.0.0 这类带通配符的 IP 和特定的 IP 地址，
```

File: content/zh/docs/reference/command-line-tools-reference/kube-scheduler.md [hunk 19]
```diff
@@ -786,22 +850,38 @@ Kubernetes API 服务器的地址（覆盖 kubeconfig 中的任何值）。
...
+并且它避免等待内核释放处于TIME_WAIT状态的套接字。
```
Comment (tengqm) [inline content/zh/docs/reference/command-line-tools-reference/kube-scheduler.md:869]:
```suggestion
并且它避免等待内核释放处于 TIME_WAIT 状态的套接字。
```

File: content/zh/docs/reference/command-line-tools-reference/kube-scheduler.md [hunk 20]
```diff
@@ -853,22 +933,25 @@ DEPRECATED: the namespace where policy ConfigMap is located. The kube-system nam
 <tr>
 <td></td><td style="line-height: 130%; word-wrap: break-word;">
 <!--
-DEPRECATED: the port on which to serve HTTP insecurely without authentication and authorization. If 0, don't serve plain HTTP at all. See --secure-port instead.
+DEPRECATED: the port on which to serve HTTP insecurely without authentication and authorization. If 0, don't serve plain HTTP at all. See --secure-port instead. This parameter is ignored if a config file is specified in --config.
 -->
 已弃用: 在没有身份验证和授权的情况下不安全地为 HTTP 服务的端口。
-如果为0，则根本不提供 HTTP。请参见--secure-port。
+如果为0，则根本不提供 HTTP。请参见 --secure-port。
```
Comment (tengqm) [inline content/zh/docs/reference/command-line-tools-reference/kube-scheduler.md:939]:
```suggestion
如果为 0，则根本不提供 HTTP。请参见 --secure-port。
```

File: content/zh/docs/reference/command-line-tools-reference/kube-scheduler.md [hunk 21]
```diff
@@ -937,15 +1023,17 @@ List of request headers to inspect for usernames. X-Remote-User is common.
...
+默认值："default-scheduler"</td>
 </tr>
 <tr>
 <td></td><td style="line-height: 130%; word-wrap: break-word;">
 <!-- 
-DEPRECATED: name of the scheduler, used to select which pods will be processed by this scheduler, based on pod's "spec.schedulerName".
+DEPRECATED: name of the scheduler, used to select which pods will be processed by this scheduler, based on pod's &quot;spec.schedulerName&quot;. This parameter is ignored if a config file is specified in --config.
 -->
-已弃用: 调度器名称，用于根据 Pod 的 "spec.schedulerName" 选择此
+已弃用: 调度器名称，用于根据 Pod 的 &quot;spec.schedulerName&quot; 选择此
```
Comment (tengqm) [inline content/zh/docs/reference/command-line-tools-reference/kube-scheduler.md:1034]: 引号不必转义。

File: content/zh/docs/reference/command-line-tools-reference/kube-scheduler.md [hunk 22]
```diff
@@ -1124,7 +1215,7 @@ Print version information and quit
 </tr>
 
 <tr>
-<td colspan="2">--vmodule moduleSpec</td>
+<td colspan="2">--vmodule &lt;comma-separated 'pattern=N' settings&gt;</td>
```
Comment (tengqm) [inline content/zh/docs/reference/command-line-tools-reference/kube-scheduler.md:1218]:
```suggestion
<td colspan="2">--vmodule &lt;<!--comma-separated 'pattern=N' settings-->逗号分隔的 '模式=N' 配置列表&gt;</td>
```

File: content/zh/docs/reference/command-line-tools-reference/kube-scheduler.md [hunk 23]
```diff
@@ -1071,18 +1161,20 @@ File containing the default x509 private key matching --tls-cert-file.
...
+A pair of x509 certificate and private key file paths, optionally suffixed with a list of domain patterns which are fully qualified domain names, possibly with prefixed wildcard segments. The domain patterns also allow IP addresses, but IPs should only be used if the apiserver has visibility to the IP address requested by a client. If no domain patterns are provided, the names of the certificate are extracted. Non-wildcard matches trump over wildcard matches, explicit domain patterns trump over extracted names. For multiple key/certificate pairs, use the --tls-sni-cert-key multiple times. Examples: &quot;example.crt,example.key&quot; or &quot;foo.crt,foo.key:*.foo.com,foo.com&quot;.
 -->
-一对 x509 证书和私钥文件路径，可选地后缀为完全限定域名的域模式列表，
-并可能带有前缀的通配符段。如果未提供域名模式，则提取证书名称。
-非通配符匹配优先于通配符匹配，显式域名模式优先于提取而来的名称。
+一对 x509 证书和私钥文件路径，可选择以 域名匹配 和 包括完整域名 的列表为后缀
```
Comment (tengqm) [inline content/zh/docs/reference/command-line-tools-reference/kube-scheduler.md:1171]:
```suggestion
一对 x509 证书和私钥文件路径，也可以包含由全限定域名构成的域名模式列表作为后缀，
```

File: content/zh/docs/reference/command-line-tools-reference/kube-scheduler.md [hunk 24]
```diff
@@ -1071,18 +1161,20 @@ File containing the default x509 private key matching --tls-cert-file.
...
+并可能带有前缀的通配符段。域名匹配还允许是 IP 地址，
+但是只有当apiserver对客户端请求的IP地址有可见性时，才能使用IP。
```
Comment (tengqm) [inline content/zh/docs/reference/command-line-tools-reference/kube-scheduler.md:1173]:
```suggestion
但是只有当 apiserver 在客户端请求的 IP 地址上可见时，才能使用 IP。
```

File: content/zh/docs/reference/command-line-tools-reference/kube-scheduler.md [hunk 25]
```diff
@@ -1071,18 +1161,20 @@ File containing the default x509 private key matching --tls-cert-file.
...
+如果未提供域名匹配，则提取证书名称。
```
Comment (tengqm) [inline content/zh/docs/reference/command-line-tools-reference/kube-scheduler.md:1174]:
```suggestion
如果未提供域名匹配模式，则提取证书名称。
```

File: content/zh/docs/reference/command-line-tools-reference/kube-scheduler.md [hunk 26]
```diff
@@ -1124,14 +1216,14 @@ Print version information and quit
 </tr>
 
 <tr>
-<td colspan="2">--vmodule moduleSpec</td>
+<td colspan="2">--vmodule &lt;<!--comma-separated 'pattern=N' settings-->逗号分隔的 '匹配=N' 配置列表&gt;</td>
```
Comment (tengqm) [inline content/zh/docs/reference/command-line-tools-reference/kube-scheduler.md:1219]:
```suggestion
<td colspan="2">--vmodule &lt;<!--comma-separated 'pattern=N' settings-->逗号分隔的 '模式=N' 配置列表&gt;</td>
```

File: content/zh/docs/reference/command-line-tools-reference/kube-scheduler.md [hunk 27]
```diff
@@ -1124,14 +1216,14 @@ Print version information and quit
...
 </tr>
 <tr>
 <td></td><td style="line-height: 130%; word-wrap: break-word;">
 <!--
 comma-separated list of pattern=N settings for file-filtered logging
 -->
-以逗号分隔的 pattern=N 设置列表，用于文件过滤的日志记录。
+以逗号分隔的 匹配=N 设置列表，用于文件过滤的日志记录。
```
Comment (tengqm) [inline content/zh/docs/reference/command-line-tools-reference/kube-scheduler.md:1226]:
```suggestion
以逗号分隔的'模式=N'设置列表，用于基于文件过滤日志记录。
```

File: content/zh/docs/reference/command-line-tools-reference/kube-scheduler.md [hunk 28]
```diff
@@ -786,22 +851,38 @@ Kubernetes API 服务器的地址（覆盖 kubeconfig 中的任何值）。
...
+<tr>
+<td colspan="2">--permit-address-sharing</td>
+</tr>
+<tr>
+<td></td><td style="line-height: 130%; word-wrap: break-word;">
+<!--
+If true, SO_REUSEADDR will be used when binding the port. This allows binding to wildcard IPs like 0.0.0.0 and specific IPs in parallel, and it avoids waiting for the kernel to release sockets in TIME_WAIT state. [default=false]
+-->
+如果为 true，在绑定端口时将使用 SO_REUSEADDR。
+这将允许通配符IP比如 0.0.0.0 和指定 ip 一起并行绑定，
```
Comment (tengqm) [inline content/zh/docs/reference/command-line-tools-reference/kube-scheduler.md:869]: please check previous comment to this:

```suggestion
这将允许同时绑定诸如 0.0.0.0 这类通配符 IP和特定 IP，
```

File: content/zh/docs/reference/command-line-tools-reference/kube-scheduler.md [hunk 29]
```diff
@@ -853,22 +934,25 @@ DEPRECATED: the namespace where policy ConfigMap is located. The kube-system nam
 <tr>
 <td></td><td style="line-height: 130%; word-wrap: break-word;">
 <!--
-DEPRECATED: the port on which to serve HTTP insecurely without authentication and authorization. If 0, don't serve plain HTTP at all. See --secure-port instead.
+DEPRECATED: the port on which to serve HTTP insecurely without authentication and authorization. If 0, don't serve plain HTTP at all. See --secure-port instead. This parameter is ignored if a config file is specified in --config.
 -->
-已弃用: 在没有身份验证和授权的情况下不安全地为 HTTP 服务的端口。
-如果为0，则根本不提供 HTTP。请参见--secure-port。
+已弃用： 在没有身份验证和授权的情况下不安全地为 HTTP 服务的端口。
```
Comment (tengqm) [inline content/zh/docs/reference/command-line-tools-reference/kube-scheduler.md:939]:
```suggestion
已弃用： 在没有身份验证和鉴权的情况下不安全地为 HTTP 服务的端口。
```

File: content/zh/docs/reference/command-line-tools-reference/kube-scheduler.md [hunk 30]
```diff
@@ -1071,18 +1161,20 @@ File containing the default x509 private key matching --tls-cert-file.
...
+A pair of x509 certificate and private key file paths, optionally suffixed with a list of domain patterns which are fully qualified domain names, possibly with prefixed wildcard segments. The domain patterns also allow IP addresses, but IPs should only be used if the apiserver has visibility to the IP address requested by a client. If no domain patterns are provided, the names of the certificate are extracted. Non-wildcard matches trump over wildcard matches, explicit domain patterns trump over extracted names. For multiple key/certificate pairs, use the --tls-sni-cert-key multiple times. Examples: &quot;example.crt,example.key&quot; or &quot;foo.crt,foo.key:*.foo.com,foo.com&quot;.
 -->
-一对 x509 证书和私钥文件路径，可选地后缀为完全限定域名的域模式列表，
-并可能带有前缀的通配符段。如果未提供域名模式，则提取证书名称。
-非通配符匹配优先于通配符匹配，显式域名模式优先于提取而来的名称。
+一对 x509 证书和私钥文件路径，也可以包含由全限定域名构成的域名模式列表作为后缀，
+并可能带有前缀的通配符段。域名匹配还允许是 IP 地址，
+但是只有当 apiserver 对客户端请求的IP地址可见时，才能使用 IP。
```
Comment (tengqm) [inline content/zh/docs/reference/command-line-tools-reference/kube-scheduler.md:1173]:
```suggestion
但是只有当 apiserver 对客户端请求的 IP 地址可见时，才能使用 IP。
```

File: content/zh/docs/reference/command-line-tools-reference/kube-scheduler.md [hunk 31]
```diff
@@ -1124,14 +1216,14 @@ Print version information and quit
...
-<td colspan="2">--vmodule moduleSpec</td>
+<td colspan="2">--vmodule &lt;<!--comma-separated 'pattern=N' settings-->逗号分隔的 '模式=N' 配置列表&gt;</td>
 </tr>
 <tr>
 <td></td><td style="line-height: 130%; word-wrap: break-word;">
 <!--
 comma-separated list of pattern=N settings for file-filtered logging
 -->
-以逗号分隔的 pattern=N 设置列表，用于文件过滤的日志记录。
+以逗号分隔的 模式=N 设置列表，用于文件过滤的日志记录。
```
Comment (tengqm) [inline content/zh/docs/reference/command-line-tools-reference/kube-scheduler.md:1226]:
```suggestion
以逗号分隔的‘模式=N’设置列表，用于文件过滤的日志记录。
```

**Comment by tengqm** [comment]
Thanks.
/approve

**Pull request merged**